src/files/MMTFFile.cpp

```diff
@@ -6,18 +6,16 @@
 #include "chemfiles/utils.hpp"
 #include "chemfiles/files/MMTFFile.hpp"
 
+#include <mmtf.hpp>
+
 using namespace chemfiles;
 
 MMTFFile::MMTFFile(std::string filename, File::Mode mode):
-  File(std::move(filename), mode), handle_(MMTF_container_new()) {
-
-    if (handle_ == nullptr) {
-        throw memory_error("Could not allocate a MMTF container");
-    }
+  File(std::move(filename), mode), handle_() {
 
     if (mode == File::READ) {
```

Comment:
Since write/append mode are not supported, could you throw an error in these cases ?

Reply:
I plan on adding these eventually, but in the meantime, if we are merging this PR I can.

Reply:
That's up to you! I am fine with merging this as it is, or waiting for write support

Reply:
So, what are we doing? Merging this as it is or waiting for write support?

Reply:
Answering to myself: I think I'll merge this (I just rebased on top of master) because the previous implementation is blocking me on another branch =)